src/rpm/package.rs

```diff
@@ -294,7 +312,7 @@ impl RPMPackageMetadata {
         Ok(())
     }
 
-    pub fn get_digests(&self) -> Result<Digests, RPMError> {
+    fn get_digests(&self) -> Result<Digests, RPMError> {
```

Comment:
Problem: all these digests are / have to be calculated in different places, and this `Digests` struct is shared with the creation code, so we can't just add them here.  Need to rework this a bit so to reduce churn let's not expose it yet.

Reply:
Yup, I'm just stating that as a practical matter it's hard to share this in both places.  I could make the creation code return a tuple instead and free this up to be user-facing?